rs/sns/testing/src/main.rs

```diff
@@ -0,0 +1,26 @@
+use clap::Parser;
+use ic_sns_testing::pocket_ic::bootstrap_nns;
+use pocket_ic::PocketIcBuilder;
+use reqwest::Url;
+
+#[derive(Debug, Parser)]
+struct SnsTestingOpts {
+    #[arg(long)]
+    server_url: Url,
+}
+
+#[tokio::main]
+async fn main() {
+    let opts = SnsTestingOpts::parse();
+    let mut pocket_ic = PocketIcBuilder::new()
+        .with_server_url(opts.server_url)
```

Comment:
Is the URL mandatory? What happens if it's not specified? 

For the scenario in which the URL is provided by the user who started the PocketIC instance themself, can we add a validation step that would check that the most important conditions are met for that instance? In particular, that we have the expected subnets and NNS canisters installed.

Reply:
If the above is too involved to implement, maybe we could at least do as follows:
1. Check if the network is "empty," and if so, bootstrap all needed canisters.
2. Otherwise, fall back to checking if the user already has everything needed.

Reply:
> Is the URL mandatory?

Yes, this is mandatory. The motivation behind using an externally provided PocketIC server URL is that the server launched by the `PocketIc` client always uses the default 60 seconds TTL which is not enough to complete and interact with the scenario.

Perhaps, it might be a good idea to make TTL configurable on the `PocketIc` client side, but for now, the server URL is mandatory

Reply:
I actually like the idea to leave TTL configuration to the user, as it's orthogonal to the sns-testing scenario.